main.go

```diff
@@ -129,6 +129,15 @@ func main() {
 
 	daemonCommand.PersistentFlags().String(f.Service.Deployer.Installer.Configurer.File.Path, "", "Path to values file.")
 
+	daemonCommand.PersistentFlags().String(f.Service.Deployer.Installer.Configurer.Configmap.Address, "", "Address of Kubernetes API server.")
+	daemonCommand.PersistentFlags().String(f.Service.Deployer.Installer.Configurer.Configmap.CAFilePath, "", "Path to CA file for Kubernetes API server.")
+	daemonCommand.PersistentFlags().String(f.Service.Deployer.Installer.Configurer.Configmap.CrtFilePath, "", "Path to crt file for Kubernetes API server.")
+	daemonCommand.PersistentFlags().Bool(f.Service.Deployer.Installer.Configurer.Configmap.InCluster, true, "Whether draughtsman is running inside a Kubernetes cluster.")
+	daemonCommand.PersistentFlags().String(f.Service.Deployer.Installer.Configurer.Configmap.Key, "values", "Key in configmap holding values data.")
+	daemonCommand.PersistentFlags().String(f.Service.Deployer.Installer.Configurer.Configmap.KeyFilePath, "", "Path to key file for Kubernetes API server.")
+	daemonCommand.PersistentFlags().String(f.Service.Deployer.Installer.Configurer.Configmap.Name, "draughtsman", "Name of configmap holding values data.")
+	daemonCommand.PersistentFlags().String(f.Service.Deployer.Installer.Configurer.Configmap.Namespace, "default", "Namespace of configmap holding values data.")
```

Comment:
I'm not a fan either, it got away from me. Planning on fixing up properly next, see https://github.com/giantswarm/draughtsman/issues/14